test/e2e/tests/request-queuing/ui.spec.js

```diff
@@ -61,7 +61,7 @@ async function openDappAndSwitchChain(driver, dappUrl, chainId) {
           (permission) =>
             permission.parentCapability === PermissionNames.permittedChains,
```

Comment:
How will this test not fail given that the referenced permission doesn't exist anymore?